web/packages/shared/components/Notification/Notification.tsx

```diff
@@ -171,13 +179,48 @@ function getRenderedContent(
           color="text.secondary"
           css={longerTextCss}
         >
+          {content.list && <ListRenderer list={content.list} />}
           {content.description}
         </Text>
+        {content.link && (
+          <Link href={content.link.href} target="_blank">
+            {content.link.text}
```

Comment:
I just noticed that the link has different font size compared to the notification text size.

Reply:
I mistakenly put `<Link>` outside of `<Text>` which resulted in two issues: 
- font size was different,
- link was visible even when on a collapsed notification.

Thanks for catching this!